### PR TITLE
DEVOPS-5119 Better actions readme

### PR DESCRIPTION
**Pull request by glothriel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -175,13 +225,13 @@ Example usage
 
 ## Inclint (internal)
 
-This action should not probably used by workflows, rather by other actions, that implement linter tests for given languages. The action allows storing amount of linter errors for given branch in AWS S3 and comparing amount of linter errors results during PRs. It ensures, that amount of linter errors is incrementally decreased with time, by enforcing small improvement with every PR.
+This action should not probably used directly by workflows, rather by other actions, that implement linter tests for given languages. The action allows storing amount of linter errors for given branch in AWS S3 and comparing amount of linter errors results during PRs. It ensures, that amount of linter errors is incrementally decreased with time, by enforcing small improvement with every PR.
```
Comment (Decard6) [inline README.md:228]: https://dictionary.cambridge.org/grammar/british-grammar/amount-of-number-of-or-quantity-of

:D

File: README.md [hunk 2]
```diff
@@ -155,13 +205,13 @@ Example usage:
 
 ## Coverage (internal)
 
-This action should not probably used by workflows, rather by other actions, that implement unit tests for given languages. The action allows storing coverge of given branch in AWS S3 and comparing coverage results during PRs. It ensures, that coverage is incrementally improved with time, by enforcing small improvement with every PR, up to given threshold (currently 80%).
+This action should not probably used directly by workflows, rather by other actions, that implement unit tests for given languages. The action allows storing coverge of given branch in AWS S3 and comparing coverage results during PRs. It ensures, that coverage is incrementally improved with time, by enforcing small improvement with every PR, up to given threshold (currently 80%).
```
Comment (Decard6) [inline README.md:208]: `with time` -> `over time`

Fortunately it doesn't really matter. Approve time ^

**Review by Decard6 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (glothriel) [reply README.md:208]: Thanks, fixed

File: README.md (hunk 1 above)
Comment (glothriel) [reply README.md:228]: Thanks, fixed

**Pull request merged**